evm/bridge-token-factory/contracts/BridgeTokenFactoryWormhole.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.24;
+
+import {BridgeTokenFactory} from "./BridgeTokenFactory.sol";
+import "./Borsh.sol";
+
+interface IWormhole {
+    function publishMessage(
+        uint32 nonce,
+        bytes memory payload,
+        uint8 consistencyLevel
+    ) external payable returns (uint64 sequence);
+}
+
+contract BridgeTokenFactoryWormhole is BridgeTokenFactory {
+    IWormhole private _wormhole;
+    uint8 private _consistencyLevel;
```

Comment:
What about make it const or add some recommendation for this value: https://wormhole.com/docs/build/reference/consistency-levels/

Reply:
This is chain-specific so has to be parametrized